Sources/VaporForms/Fieldset.swift

```diff
@@ -56,12 +56,20 @@ public struct Fieldset {
     fields.forEach { fieldName, fieldDefinition in
       // For each field, see if there's a matching value in the Content
       // Fail if no matching value for a required field
-      guard let value = content[fieldName] as? Node else {
-        if requiredFieldNames.contains(fieldName) {
-          errors[fieldName].append(.requiredMissing)
+        var value: Node
+        if let nodeValue = content[fieldName] as? Node {
+            value = nodeValue
+        } else if let multipart = content[fieldName] as? Multipart,
+            case let .input(string) = multipart {
+            value = Node(string)
+        } else {
```

Comment:
this section could be more neatly styled as
```swift
else if
  let multipart = content[fieldName] as? Multipart,
  case let .input(string) = multipart
{
  value = Node(string)
} else {
```
to better fit with the rest of the library